test/srv6-compose/src/nf.rs

```diff
@@ -6,24 +7,28 @@ use netbricks::scheduler::Scheduler;
 fn tcp_ipv6_nf<T: 'static + Batch<Header = Ipv6Header>>(parent: T) -> CompositionBatch {
     parent
         .map(box |pkt| {
-            println!("IPv6-normcore hdr {}", pkt.get_header());
+            println!(
+                "IPv6-normcore hdr {}",
+                format!("{}", pkt.get_header()).red()
+            );
         })
         .parse::<TcpHeader<Ipv6Header>>()
         .map(box |pkt| {
-            println!("TCP header {}", pkt.get_header());
+            println!("TCP header {}", format!("{}", pkt.get_header()).red());
         })
         .compose()
 }
 
+#[inline]
 fn tcp_sr_nf<T: 'static + Batch<Header = Ipv6Header>>(parent: T) -> CompositionBatch {
     parent
-        .parse::<SegmentRoutingHeader<Ipv6Header>>()
+        .parse::<SegmentRoutingHeader<Ipv6Header, U_>>()
```

Comment:
Hmm, instead of this, would this type alias work?

`type SegmentRoutingHeader<Ipv6Header> = SegmentRoutingHeader<Ipv6Header, U0>`

Reply:
Can't name it the same, but I did give it a new shorter name that we can use.